scripts/verifyProposal.ts

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "ethers";
+
+import { proposeVIP } from "../src/transactions";
+
+export async function getTransactionData(vip: number, txHash: string, networkId: number) {
```

Comment:
AFAIK Include a mainnet test fork for the VIP and get the function tested. Instead of mocking the behaviour with `sinon`.

Reply:
We would require sinon to mock the data because in tests we are hardcoding calldata from bsc explorer and creating similar input with the help of sinon and verifying the functionality of build proposal